website/docs/guides/federation.html.markdown

```diff
@@ -40,33 +40,33 @@ Remember to check out the documentation for the resource you are interested in f
 
 The following Resources are available to use with Federation:
 
- * Tier-0 Gateway [nsxt_policy_tier0_gateway](https://www.terraform.io/docs/providers/nsxt/r/policy_tier0_gateway.html)
- * Tier-0 Interface [nsxt_policy_tier0_interface](https://www.terraform.io/docs/providers/nsxt/r/policy_tier0_interface.html)
- * Tier-1 Gateway [nsxt_policy_tier1_gateway](https://www.terraform.io/docs/providers/nsxt/r/policy_tier1_gateway.html)
- * Tier-1 Interface [nsxt_policy_tier1_interface](https://www.terraform.io/docs/providers/nsxt/r/policy_tier1_interface.html)
- * Segment [nsxt_policy_segment](https://www.terraform.io/docs/providers/nsxt/r/policy_segment.html)
- * VLAN Segment [nsxt_policy_vlan_segment](https://www.terraform.io/docs/providers/nsxt/r/policy_vlan_segment.html)
- * Group [nsxt_policy_group](https://www.terraform.io/docs/providers/nsxt/r/policy_group.html)
- * Service [nsxt_policy_service](https://www.terraform.io/docs/providers/nsxt/r/policy_service.html)
- * DFW Security Policy [nsxt_policy_security_policy](https://www.terraform.io/docs/providers/nsxt/r/policy_security_policy.html)
- * Gateway Policy [nsxt_policy_gateway_policy](https://www.terraform.io/docs/providers/nsxt/r/policy_gateway_policy.html)
- * NAT Rule [nsxt_policy_nat_rule](https://www.terraform.io/docs/providers/nsxt/r/policy_nat_rule.html)
+ * Tier-0 Gateway [nsxt_policy_tier0_gateway](https://www.terraform.io/docs/providers/nsxt/r/policy_tier0_gateway)
+ * Tier-0 Interface [nsxt_policy_tier0_interface](https://www.terraform.io/docs/providers/nsxt/r/policy_tier0_interface)
```

Comment:
should be https://www.terraform.io/docs/providers/nsxt/r/policy_tier0_gateway_interface, same below for tier1